pkg/traces/config.go

```diff
@@ -323,8 +323,8 @@ func exporter(rwCfg RemoteWriteConfig) (map[string]interface{}, error) {
 	}
 
 	compression := rwCfg.Compression
-	if compression == compressionNone {
-		compression = ""
+	if compression == "" {
```

Comment:
Was this an existing bug?

Reply:
Everything in pkg/traces was changed by @mapno, but I think what happened here is we needed to change our logic after the OTel upgrade to get things to pass again